superset/config.py

```diff
@@ -645,8 +645,10 @@ class D3Format(TypedDict, total=False):
 THUMBNAIL_CHART_DIGEST_FUNC: Optional[Callable[[Slice, ExecutorType, str], str]] = None
 
 THUMBNAIL_CACHE_CONFIG: CacheConfig = {
-    "CACHE_TYPE": "NullCache",
-    "CACHE_NO_NULL_WARNING": True,
+    'CACHE_TYPE': 'redis',
+    'CACHE_DEFAULT_TIMEOUT': 24*60*60*7,
```

Comment:
Could you use the `timedelta` instead ([example](https://github.com/apache/superset/blob/b8a3eeffdbfd6db852510ac409e4852fbac2f52c/superset/config.py#L952)) to improve readability.